lnbits/app.py

```diff
@@ -341,10 +394,14 @@ async def listeners():
         loop.create_task(catch_everything_and_restart(invoice_listener))
         loop.create_task(catch_everything_and_restart(internal_invoice_listener))
         await register_task_listeners()
+        await register_watchdog()
```

Comment:
- `register_watchdog()` should not be called anymore

Reply:
commented out